app/views/spree/admin/shared/_product_sub_menu.html.haml

```diff
@@ -1,8 +1,10 @@
 - content_for :sub_menu do
   %ul#sub_nav.inline-menu
-    = tab :products, match_path: '/products'
+    = tab :products
     = tab :properties
     = tab :variant_overrides, url: main_app.admin_inventory_path, match_path: '/inventory'
     = tab :import, url: main_app.admin_product_import_path, match_path: '/product_import'
     - if feature?(:new_products_page, spree_current_user)
-      = tab :new_products, url: main_app.admin_new_products_path, match_path: '/new_products'
+      = tab :new_products, url: main_app.admin_new_products_path
```

Comment:
This one should be deleted I guess ;)

Reply:
Yes I've been meaning to ask!

Reply:
--> https://github.com/openfoodfoundation/openfoodnetwork/issues/11153